rest_framework/__init__.py

```diff
@@ -35,3 +35,7 @@ class RemovedInDRF313Warning(DeprecationWarning):
 
 class RemovedInDRF314Warning(PendingDeprecationWarning):
     pass
```

Comment:
I think this and the above `RemovedInDRF313Warning` can be removed? I cannot find any other references.

Reply:
I kept it as per the comments in https://github.com/encode/django-rest-framework/pull/8589.

Reply:
That argument doesn't really convince me, DRF has never kept around the old warning classes for previous versions, e.g. `RemovedInDRF312Warning` was removed in d45e0005f32bc246a49b209836d233f3d23d77b0. Similarly, Django does not keep around old warning classes "just in case".

Reply:
I'd rather leave that decision up to a maintainer. I wasn't a part of the original discussion so it'd be disingenuous for me to say that by convincing me, the past argument is moot.

Reply:
Sorry to have missed this thread!

I was assuming that `RemovedInDRF313Warning` would be deleted with the 3.14 release.
(My point was only that deleting an exception class in a minor release would constitute a breaking change)

Reply:
CC @adamchainz - put out a PR to delete the now-unused warnings: https://github.com/encode/django-rest-framework/pull/8664